superset/sql_lab.py

```diff
@@ -73,7 +73,6 @@
 SQLLAB_HARD_TIMEOUT = SQLLAB_TIMEOUT + 60
 SQL_MAX_ROW = config["SQL_MAX_ROW"]
 SQLLAB_CTAS_NO_LIMIT = config["SQLLAB_CTAS_NO_LIMIT"]
-SQL_QUERY_MUTATOR = config["SQL_QUERY_MUTATOR"]
```

Comment:
Removing this configuration variable would be deemed a breaking change.

Reply:
Oh I'm not removing this config flag, just centralizing it's usage into the Database model -> 
https://github.com/apache/superset/pull/27943/files#diff-462c756528595a2be4a0a3c3063077f984384c7858ecdb5bd3e9603163c3678bR590

Reply:
I'll go ahead and remove the labels as this is not a breaking change